examples/taxi-ride-akka/README.md

```diff
@@ -0,0 +1,95 @@
+## Akka implementation of taxi ride application
+
+This example demonstrates how `taxi-ride` application initially implemented in flink can be implemented using Akka.
+
+The implementation combines two streams `TaxiRide` and `TaxiFare` and for each rideID calculates
+a fare for a given ride 
+
+In the case of Flink we used `RichCoFlatMapFunction` to combine the two streams, here
+we are using `RideShare` actor for the same purpose. If we compare implementation of the actor 
+with the implementation of Flink's `RichCoFlatMapFunction`, we will see that they are very similar.
+The missing part here is persistence. While Flink implementation provides automatic snapshotting, Akka
+requires implementation of an explicit persistence support, which is not currently supported
```

Comment:
Why is this not supported? Do you mean we do not install a persistent journal? It would be great if that can be done in this example. (is there a simple option that would work on a distributed file system, using a pvc?) Otherwise it's not a great comparison.

Reply:
Please add persistence for jdbc, with a postgres helm chart for instance.

Reply:
Done